src/Sfa.Tl.Matching.Data/Models/RoutePath.cs

```diff
@@ -7,14 +7,14 @@ public class RoutePath
     {
         public Guid Id { get; set; }
         public Guid CourseId { get; set; }
-        public string Route { get; set; }
-        public string Path { get; set; }
+        public int RoutePathLookupId { get; set; }
         public string Summary { get; set; }
         public string Keywords { get; set; }
         public DateTime? CreatedOn { get; set; }
         public DateTime? ModifiedOn { get; set; }
 
         public virtual Course Course { get; set; }
         public virtual ICollection<IndustryPlacement> IndustryPlacement { get; set; }
+        public virtual ICollection<RoutePathLookup> RoutePathLookup { get; set; }
```

Comment:
I think this is just public virtual RoutePathLookup RoutePathLookup { get; set; }

Reply:
Agreed - this has been fixed and pushed.